src/platforms/esp32/components/avm_builtins/i2c_driver.c

```diff
@@ -99,9 +106,36 @@ Context *i2c_driver_create_port(GlobalContext *global, term opts)
     struct I2CData *i2c_data = calloc(1, sizeof(struct I2CData));
     i2c_data->transmitting_pid = term_invalid_term();
 
-    term scl_io_num_term = interop_proplist_get_value(opts, SCL_IO_NUM_ATOM);
-    term sda_io_num_term = interop_proplist_get_value(opts, SDA_IO_NUM_ATOM);
-    term clock_hz_term = interop_proplist_get_value(opts, I2C_CLOCK_HZ_ATOM);
+    term scl_io_num_term = interop_kv_get_value_default(
```

Comment:
As soon as #415 gets merged, we can replace the `interop_kv_get_value_default` which is the same, but more compact.